CHANGELOG.md

```diff
@@ -50,19 +53,19 @@ TypeVarTuple was implemented by Jared Hance and Ivan Levkivskyi over several myp
 
 Changes included in this release:
 
- * Fix handling of tuple type context with unpacks (Ivan Levkivskyi, PR [16444](https://github.com/python/mypy/pull/16444))
```

Comment:
Can you undo these changes to keep the PR small?

Reply:
Gnnnn. Sorry I missed that VSCode reformatted the file ...